fixtures/transformers/transformer.js

```diff
@@ -0,0 +1,20 @@
+exports.transformers = [
+  {
+    name: "Test transformations with transfromer test id",
+    transform: (body, req, res) => {
+      console.log("Resource Path", req.method, req.url);
+      console.log(req.url);
+      const transformedTestId = "test-with-transformer";
+
+      if (req.url === `/helloworld/${transformedTestId}`) {
+        console.log("resource path has matched");
```

Comment:
Does this line still need to be here?

Reply:
This gives information confirming that the url path has been matched. In a transformer, some log message can be expected, but in this case removing this log won't affect our test and we already have at least one log.